udata_front/theme/gouvfr/templates/user/base.html

```diff
@@ -37,7 +36,14 @@
     >
         {{ _("User's admin profile") }}
     </a>
-    {{ delete_btn(user) }}
+    <div class="fr-col-auto vuejs">
+        <delete-user-button
```

Comment:
I think the component should be added to index.ts to allow it to be used like this

Reply:
It is, you can see it in the changed files